### ISSUE TITLE
Focus Maintains Context

### ISSUE THREAD

**Issue by chriscm2006** [dequelabs/axe-android#18]
A focus change should not result in a context change, such as focusing another control.

**Effected Technology**: TalkBack 
**Type**: Serious - WCAG 2.1 A - Violation
**Success Criteria**: [3.2.1](https://www.w3.org/TR/WCAG21/#on-focus)

## User Story

As a TalkBack User I am using an Activity that utilizes constraint layouts heavily.

### Inaccessible

1. Open an `Activity`
2. Move `Accessibility Focus` to a `View(1)`
3. This `View` initiates an `Accessibility Focus` event to `View(2)`

The Change of Context in 3 is in violation of WCAG 3.2.1.

### Accessible

1. Open an `Activity`
2. Move `Accessibility Focus` to a `View`
3. Focus remains on `View(1)`
